MetricsLibraryEditor/ExternalMetrics.js

```diff
@@ -18,8 +18,8 @@ app.configure(function () {
     app.use(express.bodyParser());
 	app.use(express.cookieParser('Test'));
 	app.use(express.session());
-    app.use(app.router);
-
+  app.use(app.router);
+  app.use(express.static('C:\\Program Files\\Qlik\\Sense\\ServiceDispatcher\\Node\\Metrics-Library-Editor\\public'));
```

Comment:
What's a good way to 'include' a public folder so I can ref all the style and .js files without having to include each one manually? This seems hacky. Also, the installer would need to include the `public` folder.